SafeExecute/hook.cpp

```diff
@@ -153,11 +155,21 @@ BOOL WINAPI MoveFileW_Hook(
     LPCTSTR lpNewFileName
 ) {
     PreHook(1, "MoveFileW");
-    // MoveFile Hook
+    // MoveFileW Hook
     ExitProcess(1);
     return orig_MoveFileW(lpExistingFileName, lpNewFileName);
 }
 
+BOOL WINAPI MoveFileA_Hook(
+    LPCTSTR lpExistingFileName,
+    LPCTSTR lpNewFileName
+) {
+    PreHook(1, "MoveFileA");
+    // MoveFileA Hook
```

Comment:
Can you write interactive mode's contents here?
something like, asking below by a msgbox
```
This executable is trying to rename lpExistingFileName to lpNewFileName
Continue execution?
```
and ExitProcess if IDNO.